src/components/login/prompt/ArticleLoginPrompt.js

```diff
@@ -49,13 +50,14 @@ export default function ArticleLoginPrompt({ articleId }) {
     );
   }
 
-  // Logged in, library does not have access
-  if (!isLoading && !data?.infomedia?.article) {
+  //AUTHENTICATED AND NO ACCESS either because, we couldnt fetch article (shoudl we show error instead?)
+  // OR bc user doesnt have access rights
+  // --> Show library name and explain how to obtain access
+  if (!isLoading && (!data?.infomedia?.article || !user.rights.infomedia)) {
```

Comment:
Lille ting, men du kan genbruge hasInfomediaAccess istedet for user.rights.infomedia